src/app/series/series.component.html

```diff
@@ -31,6 +31,7 @@ <h1 class="hero-title" *ngIf="!id">Create Series</h1>
     <a routerLinkActive="active" [routerLinkActiveOptions]="{exact:true}" [routerLink]="base">Basic Info</a>
     <a routerLinkActive="active" [routerLink]="[base, 'templates']">Audio Templates</a>
     <a routerLinkActive="active" [routerLink]="[base, 'podcast']">Podcast Distribution</a>
+    <a routerLinkActive="active" [routerLink]="[base, 'feed']">Stories in Feed</a>
   </nav>
   <div class="extras">
     <a *ngIf="storyCount > 0" [routerLink]="['/search', { tab: 'stories', seriesId: id }]">{{storyCount}} {{storyNoun}} &raquo;</a>
```

Comment:
Wondering if you should remove this link, and maybe put it on your new tab somewhere.  Your tab-label could also use this count, if you want it to be a bit more descriptive.